nsv13/modules/credits/credits.dm

```diff
@@ -0,0 +1,296 @@
+var/global/datum/credits/end_credits = new
```

Comment:
This will need changing to something else but I dont know what the new correct standard is.

Reply:
a no_fire `SSCredits`, probably.

Reply:
That is probably the best way.